backend/python/app/models/comment.py

```diff
@@ -16,9 +17,8 @@ class Comment(db.Model):
         index=True,
         nullable=False,
     )
-
+    story_translation_content = db.relationship(StoryTranslationContent)
```

Comment:
Is this the equivalent of adding backref onto the story_translation_content?

Reply:
yep! also equivalent to two `back_populate` relationship attributes https://docs.sqlalchemy.org/en/14/orm/backref.html

Reply:
Just comparing to StoryTranslation / ...Content, and Story / ...Content, where it's one-to-many. In these cases, it looks like the "one" declares the relationship and the "many" recognize foreign key?
https://github.com/uwblueprint/planet-read/blob/52d12c0f384eb1b6bfefe4b865b738d1045fd8f0/backend/python/app/models/story_translation.py#L20

In this case, one StoryTContent -> many comments, right? Why does this notation still work and why define the relationship in the "many" class this time rather than the "one"?

Reply:
> Why does this notation still work

relationships can be a two way street (😉) https://docs.sqlalchemy.org/en/14/tutorial/orm_related_objects.html
> Why define the relationship in the "many" class this time rather than the "one"?

In this case I define the relationship on comment because I want immediate access to the associated story_translation_content in create_comment to change the status.

Reply:
Uhg. Starting to question establishing this relationship here since joining with story_translation_content really isnt necessary for all commenting needs. going to come back to this in the morning

Reply:
^^ Did more reading on https://docs.sqlalchemy.org/en/14/orm/loading_relationships.html, I believe SQLAlchemy to be smart enough with caching and loading that this isnt an issue.